src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -326,7 +325,7 @@ def body(self):
         # Test inbound NAT rules
         for count in range(1, 4):
             self.cmd('network lb inbound-nat-rule create {} -n rule{} --protocol tcp --frontend-port {} --backend-port {} --frontend-ip-name LoadBalancerFrontEnd'.format(lb_rg, count, count, count))
-        self.cmd('network lb inbound-nat-rule list {}'.format(lb_rg),
+        self.cmd('network lb inbound-nat-rule list -g {} --lb-name {}'.format(rg, lb),
```

Comment:
It had to do with the fact that for list it used to be '-g {} -n {}' so the pattern of '-g {} --lb-name {}' didn't hold. Since we changed that, I collapsed these references

---
In reply to: [73071998](https://github.com/Azure/azure-cli/pull/592#discussion_r73071998) [](ancestors = 73071998)